[2025-05-09T16:21:43.250+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-09T15:21:21.879216+00:00 [queued]>
[2025-05-09T16:21:43.269+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-09T15:21:21.879216+00:00 [queued]>
[2025-05-09T16:21:43.299+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-09T16:21:43.367+0100] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_clean_reviews> on 2025-05-09 15:21:21.879216+00:00
[2025-05-09T16:21:43.383+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'process_bank_reviews', 'run_clean_reviews', 'manual__2025-05-09T15:21:21.879216+00:00', '--job-id', '1884', '--raw', '--subdir', 'DAGS_FOLDER/post_processing.py', '--cfg-path', '/tmp/tmpvcpgbju6']
[2025-05-09T16:21:43.379+0100] {standard_task_runner.py:60} INFO - Started process 72523 to run task
[2025-05-09T16:21:43.386+0100] {standard_task_runner.py:88} INFO - Job 1884: Subtask run_clean_reviews
[2025-05-09T16:21:43.477+0100] {task_command.py:423} INFO - Running <TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-09T15:21:21.879216+00:00 [running]> on host DESKTOP-RRUTCI2.
[2025-05-09T16:21:43.605+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_bank_reviews' AIRFLOW_CTX_TASK_ID='run_clean_reviews' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T15:21:21.879216+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-09T15:21:21.879216+00:00'
[2025-05-09T16:21:43.609+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-09T16:21:43.611+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/airflow/dbt_project && dbt run --select clean_bank_reviews']
[2025-05-09T16:21:43.624+0100] {subprocess.py:86} INFO - Output:
[2025-05-09T16:21:50.389+0100] {subprocess.py:93} INFO - [0m15:21:50  Running with dbt=1.8.7
[2025-05-09T16:21:51.114+0100] {subprocess.py:93} INFO - [0m15:21:51  Registered adapter: postgres=1.8.2
[2025-05-09T16:21:52.532+0100] {subprocess.py:93} INFO - [0m15:21:52  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-05-09T16:21:52.533+0100] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-05-09T16:21:52.534+0100] {subprocess.py:93} INFO - - models.dbt_project.forPy
[2025-05-09T16:21:52.803+0100] {subprocess.py:93} INFO - [0m15:21:52  Found 8 models, 1 seed, 19 data tests, 2 sources, 540 macros
[2025-05-09T16:21:52.804+0100] {subprocess.py:93} INFO - [0m15:21:52
[2025-05-09T16:21:54.297+0100] {subprocess.py:93} INFO - [0m15:21:54  Concurrency: 3 threads (target='dev')
[2025-05-09T16:21:54.299+0100] {subprocess.py:93} INFO - [0m15:21:54
[2025-05-09T16:21:54.301+0100] {subprocess.py:93} INFO - [0m15:21:54  1 of 1 START sql view model public.clean_bank_reviews .......................... [RUN]
[2025-05-09T16:21:54.423+0100] {subprocess.py:93} INFO - [0m15:21:54  1 of 1 OK created sql view model public.clean_bank_reviews ..................... [[32mCREATE VIEW[0m in 0.12s]
[2025-05-09T16:21:54.435+0100] {subprocess.py:93} INFO - [0m15:21:54
[2025-05-09T16:21:54.437+0100] {subprocess.py:93} INFO - [0m15:21:54  Finished running 1 view model in 0 hours 0 minutes and 1.63 seconds (1.63s).
[2025-05-09T16:21:54.467+0100] {subprocess.py:93} INFO - [0m15:21:54
[2025-05-09T16:21:54.468+0100] {subprocess.py:93} INFO - [0m15:21:54  [32mCompleted successfully[0m
[2025-05-09T16:21:54.469+0100] {subprocess.py:93} INFO - [0m15:21:54
[2025-05-09T16:21:54.469+0100] {subprocess.py:93} INFO - [0m15:21:54  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-05-09T16:21:55.343+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-09T16:21:55.367+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_bank_reviews, task_id=run_clean_reviews, execution_date=20250509T152121, start_date=20250509T152143, end_date=20250509T152155
[2025-05-09T16:21:55.394+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-09T16:21:55.413+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
